cluster/vsphere/util.sh

```diff
@@ -344,7 +344,7 @@ function kube-up {
 
   printf "Waiting for ${KUBE_MASTER} to become available..."
   until curl --insecure --user "${KUBE_USER}:${KUBE_PASSWORD}" --max-time 5 \
-          --fail --output /dev/null --silent "https://${KUBE_MASTER_IP}/api/v1beta1/pods"; do
+          --fail --output /dev/null --silent "https://${KUBE_MASTER_IP}/api/v1beta3/pods"; do
```

Comment:
I think this curl was copy-pasted to lots of places. IMO, it needn't/shouldn't be polling pods, esp. not with a direct curl. It could use kubectl, or ping /healthz or another unversioned path.